lua/gitlab/actions/create_mr.lua

```diff
@@ -78,17 +78,20 @@ local description_popup_settings = {
   },
 }
 
-local title_input_options = {
-  position = "50%",
-  relative = "editor",
-  size = 40,
-  border = {
-    style = "rounded",
-    text = {
-      top = "Title",
+---Create input options after user settings have been merged with defaults.
+local get_title_input_options = function()
+  return {
+    position = "50%",
+    relative = "editor",
+    size = state.settings.create_mr.title_input.width,
+    border = {
+      style = state.settings.create_mr.title_input.border,
+      text = {
+        top = "Title",
+      },
     },
```

Comment:
Would prefer to just pass this state directly inside of the `add_title` function, think it's a little nicer to not have to split this up:

![Screenshot 2024-02-10 at 11 17 14 AM](https://github.com/harrisoncramer/gitlab.nvim/assets/32515581/13ee4b44-b680-4feb-b010-46fa149076f5)

Reply:
@harrisoncramer Thanks for the review. I've applied your suggestion.